src/modules/map/component.ts

```diff
@@ -224,6 +224,47 @@ export async function createMapComponent(components: {
     return result || null
   }
 
+  async function getLastEstateId() {
+    const ids = Object.values(await estates)
+      .map((estate) => Number(estate.id))
+      .sort((a, b) => (a > b ? 1 : -1))
```

Comment:
The default sort is an ascending ordered sort, I think you can remove the comparison function here.

Reply:
yeh but it converts to string first..
```js
[1,2,3,4,100].sort() // ---> [1, 100, 2, 3, 4]
```

Reply:
#javascript :exito:

Reply:
Oh, in that case, would you mind adding a clause for the `a == b` case? Just to be compliant with the requirements.

Reply:
since ids are unique there should never be the case of two equal values tho